src/ServantElm.hs

```diff
@@ -151,13 +222,14 @@ mkRequest request =
             "expect ="
               <> indent4Spaces expect,
             "body ="
-              <> indent4Spaces "Http.emptyBody",
+              <> indent4Spaces body,
```

Comment:
Why are you using `indent4Spaces` here? It causes unnecessary spacing, as shown in the generated example. Use `<+>` instead, I think.

Reply:
done